aztk/aztklib.py

```diff
@@ -4,9 +4,16 @@
 
 class Aztk:
     def __init__(self):
-        secrets_config = config.SecretsConfig()
-        secrets_config.load()
+        self.secrets_config = config.SecretsConfig()
+        self.client = None
 
+
+    def load(self):
```

Comment:
Taking this a step further, you could argue that the method should take a path to the file you want to load as a parameter.